pombola/south_africa/templates/info/_blog_sidebar.html

```diff
@@ -14,7 +14,7 @@
       {% endfor %}
     </ul>
 
-    {% if popular_posts.exists %}
+    {% if some_popular_posts %}
```

Comment:
I've just added an empty block; I think it'll very rarely be seen so probably not a big deal anyway